it/src/it/simple-with-route-flag-false/expected/openshift.yml

```diff
@@ -89,7 +89,7 @@ items:
             valueFrom:
               fieldRef:
                 fieldPath: metadata.namespace
-          image: fabric8-maven-sample-zero-config:latest
+          image: fabric8/fabric8-maven-sample-zero-config:latest
           imagePullPolicy: IfNotPresent
```

Comment:
I don't think this we should do this, Try deploying this project onto openshift, DeploymentConfigs would not get rolled. I also did the same thing and had to revert this change afterwards. In minishift image is pushed as:
```
[docker@minishift ~]$ docker images | grep spring-boot              
172.30.1.1:5000/myproject/fabric8-maven-sample-spring-boot          latest              bffc2e90d804        4 minutes ago       603 MB
172.30.1.1:5000/myproject/fabric8-maven-sample-spring-boot          <none>              dbc792415de5        4 minutes ago       603 MB
```